### PR TITLE
added Opera network checking logic for simpl. API

### PR DESCRIPTION
**Pull request by W3stside**
Opera uses a simplified, `currentProvider`-only web3 API thus needs extended logic to play nicely in main production network/geo blocking

This should extend to other APIs that are formed this way

### CODE REVIEW

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr437--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr437--dxreact.review.gnosisdev.com

**Review by Velenir — APPROVED** [review]

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr437--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr437--dxreact.review.gnosisdev.com

File: src/utils/index.ts [hunk 1]
```diff
@@ -205,14 +205,23 @@ export const provider2SVG = (providerName: ProviderName | ProviderType) => {
...
+
+    window.web3 = web3 = new Web3(web3.currentProvider)
+  }
+
   // 1.X.X API
-  if (typeof window.web3.version === 'string') {
+  if (typeof web3.version === 'string') {
     netID = await new Promise<string>((accept, reject) => {
-      window.web3.eth.net.getId((err: string, res: string) => {
+      web3.eth.net.getId((err: string, res: string) => {
```
Comment (W3stside) [inline src/utils/index.ts]: :+1:

**Pull request merged**